docs/api_v0.yml

```diff
@@ -342,8 +342,6 @@ components:
       properties:
         article:
           type: object
-          required:
-            - title
```

Comment:
An article can be created without the title param in JSON if the body markdown contains the title in the frontmatter, so this attribute is technically not required